_pages/policies/responding-to-public-disclosure-vulnerabilities.md

```diff
@@ -11,9 +11,9 @@ When someone in the public alerts us to a potential vulnerability, we need to re
 
 ## Triage the vulnerability
 
-In PagerDuty, someone is on call to be the **first responder**. This role rotates weekly.
+In order to respond quickly to reports submitted by the public, the Infrastructure team has designated a small group of folks who will monitor incoming reports and initiate response accordingly. This role has been designated **first responder** and will be assigned in [#vuln-disclosure](https://gsa-tts.slack.com/messages/vuln-disclosure/) with an @ (mention). Barring schedule conflicts, the role will rotate weekly.
```

Comment:
Yeah, this is critical. If they're not whitelisted, their mail to cloud-gov-emergency@gsa.gov will be held for moderation. If they are whitelisted, it'll go through immediately.